components/activities/activities-tab/activities-tab.tsx

```diff
@@ -84,6 +106,13 @@ export function ActivitiesTabManager<
             </div>
           )
           : <div className="empty-list-group">No live activities</div>}
+        {hasPollPermissions
```

Comment:
is there a good reason to use ternary over && ?

imo ternary always starts getting a little messy. compared to

```
hasPollPermissions && <Button> ... </Button>
```